### PR TITLE
Include contracts in the repo for use by arc-js

### PR DESCRIPTION
**Pull request by tibetsprague**
I've been playing with way too many options for including accessing contracts in arc-js and especially ones deployed to Kovan, but decided we could start with this incredibly simple option. This just includes the contracts in arc, with kovan deployment addresses, and exports a function to load the contracts. Super basic, but works for now and we just need something working that we agree to move forward with while we think through better options

### CODE REVIEW

**Comment by orenyodfat** [comment]
@tibetsprague 
-could you please share the difficulties to use our npm package deploy the contracts in kovan ?
- Could you please share the lib code which trying to do that ? 
- are you sure you need to increase the gas limit ? I succeed migrate to kovan without increasing the gas.

**Comment by tibetsprague** [comment]
@orenyodfat I am not having any difficulties right now, I was able to successfully deploy to Kovan. at one point I did need to increase the gas limit. its going to need to constantly change. in arc-js my latest PR allows you to change it through various config options, we could allow that in arc too.

File: index.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export function requireContract(contractName) {
```
Comment (dkent600) [inline index.js]: JavaScript in arc?  Hmm...  ;-)  Anyway, what is this needed for?  arc-js has it's own requireContract.  BTW, we should probably sync up on my work on https://github.com/daostack/arc-js/issues/17

File: migrations/2_deploy_DAOstack.js [hunk 2]
```diff
@@ -55,7 +55,7 @@ module.exports = async function(deployer) {
     // apparently we must wrap the first deploy call in a then to avoid
     // what seem to be race conditions during deployment
     // await deployer.deploy(GenesisScheme)
-    deployer.deploy(GenesisScheme, {gas: 6500000}).then(async function(){
+    deployer.deploy(GenesisScheme, {gas: 6900000}).then(async function(){
```
Comment (dkent600) [inline migrations/2_deploy_DAOstack.js]: As of today, migrations/2_deploy_DAOstack.js is to be moved to arc-js.  See https://github.com/daostack/daostack/pull/223.

File: index.js (hunk 1 above)
Comment (dkent600) [inline index.js]: Just to reiterate what you already know:  The arc json contracts files are not supposed to contain migration addresses.  I think you view this as potentially just a work-around.  Just a heads up that a couple things, described in other comments here, don't sync with what I'm doing in arc-js (https://github.com/daostack/arc-js/issues/17), which is currently blocked by #226.  Hopefully that will be resolved very soon.

**Pull request closed without merging**